Engine/ac/global_gui.cpp

```diff
@@ -167,6 +167,12 @@ int GetTextWidth(const char *text, int fontnum) {
   if ((fontnum < 0) || (fontnum >= game.numfonts))
     quit("!GetTextWidth: invalid font number.");
 
+  if(loaded_game_file_version < kGameVersion_360 &&
+     String_IsNullOrEmpty(text) &&
+     get_font_outline(fontnum) == FONT_OUTLINE_AUTO)
+  {
+    return game_to_data_coord(1);
```

Comment:
I don't think this fallback is necessary here, this is such a niche case, and I have doubts it was relied upon in some game's design. So I'd propose to not do this unless someone finds a game which gets broken by this fix.

Reply:
That's fine, I will remove it :)

Reply:
done, removed!